device/samples/pnp/simple_thermostat.js

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+const Protocol = require('azure-iot-device-mqtt').Mqtt;
+var ProvProtocol = require('azure-iot-provisioning-device-mqtt').Mqtt;
+
+const Client = require('azure-iot-device').Client;
+const Message = require('azure-iot-device').Message;
+const SymmetricKeySecurityClient = require('azure-iot-security-symmetric-key').SymmetricKeySecurityClient;
+const ProvisioningDeviceClient = require('azure-iot-provisioning-device').ProvisioningDeviceClient;
+
+// To see the dtdl of for this https://github.com/Azure/opendigitaltwins-dtdl
+
+// String containing Hostname, Device Id & Device Key in the following formats:
+//  'HostName=<iothub_host_name>;DeviceId=<device_id>;SharedAccessKey=<device_key>'
+var deviceConnectionString = process.env.DEVICE_CONNECTION_STRING;
+
+// DPS connection information
+var provisioningHost = process.env.PROVISIONING_HOST;
+var idScope = process.env.PROVISIONING_IDSCOPE;
+var registrationId = process.env.PROVISIONING_REGISTRATION_ID;
+var symmetricKey = process.env.PROVISIONING_SYMMETRIC_KEY;
+
+const modelId = 'dtmi:com:example:Thermostat;1';
+const telemetrySendInterval = 10000;
+let intervalToken;
+let currTemp = 1 + (Math.random() * 90);
+let targetTemp = currTemp;
+let maxTemp = currTemp+10;
+
+const commandNameReboot = 'reboot';
```

Comment:
there is no commmand `reboot` in [Thermostat.json](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json)